data_steward/bq_utils.py

```diff
@@ -79,10 +79,13 @@ def get_retraction_sandbox_dataset_id():
 
 def get_retraction_dataset_ids():
     # dataset ids from which to retract, separated by spaces ' '
+    # if retraction needs to be performed on all datasets in the project, remove this env variable
```

Comment:
Since you're updating comments for both `get_retraction_dataset_ids()` and `get_retraction_submission_folder()`, would you mind using the Pep 8 doc string format?
```
one line summary
blank line
more descriptive summary or paragraph
blank line
parameter definitions, if any
expected return values
```

Reply:
Sure! It was meant to be more like a note rather than a docstring, but I guess its better formatted as a docstring.